src/app/public/testing/popover/popover-fixture.spec.ts

```diff
@@ -0,0 +1,153 @@
+import {
+  Component
+} from '@angular/core';
+
+import {
+  ComponentFixture,
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  expect,
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+import {
+  SkyPopoverFixture
+} from './popover-fixture';
+
+import {
+  SkyPopoverTestingModule
+} from './popover-testing.module';
+
+//#region Test component
+@Component({
+  selector: 'popover-test',
+  template: `
+<button
+  class="sky-btn sky-margin-inline-compact"
+  type="button"
+  [skyPopover]="myPopover"
+  [skyPopoverAlignment]="popoverAlignment"
+  [skyPopoverPlacement]="popoverPlacement"
+  #directiveRef
+>
+  Open popover on click
+</button>
+
+<sky-popover
+  [dismissOnBlur]="dismissOnBlur"
+  [popoverTitle]="popoverTitle"
+  #myPopover
+>
+  {{popoverBody}}
+</sky-popover>
+`
+})
+class PopoverTestComponent {
+  public dismissOnBlur: boolean;
+  public popoverAlignment: string;
+  public popoverBody: string = 'popover body';
+  public popoverPlacement: string;
+  public popoverTitle: string = 'popover title';
+}
+//#endregion Test component
+
+describe('Popover fixture', () => {
+  let fixture: ComponentFixture<PopoverTestComponent>;
+  let testComponent: PopoverTestComponent;
+  let popoverFixture: SkyPopoverFixture;
+
+  //#region helpers
+  function getPopoverTriggerEl(): HTMLButtonElement {
+    return document.querySelector('.sky-btn');
+  }
+
+  function openPopover(): Promise<any> {
+    expect(popoverFixture.popoverIsVisible).toEqual(false);
+
+    let triggerEl = getPopoverTriggerEl();
+    triggerEl.click();
+    fixture.detectChanges();
+
+    return fixture.whenStable().then(() => {
+      expect(popoverFixture.popoverIsVisible).toEqual(true);
+    });
+  }
+  //#endregion
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        PopoverTestComponent
+      ],
+      imports: [
+        SkyPopoverTestingModule
+      ]
+    });
+
+    fixture = TestBed.createComponent(
+      PopoverTestComponent
+    );
+    testComponent = fixture.componentInstance;
+    fixture.detectChanges();
+    popoverFixture = new SkyPopoverFixture(fixture);
+  });
+
+  it('should not expose popover properties when hidden', async () => {
```

Comment:
Does this test need to be an `async` test? I don't see it calling any async methods...